portfolio/src/main/webapp/script.js

```diff
@@ -26,10 +26,19 @@
 //   const greetingContainer = document.getElementById('greeting-container');
 //   greetingContainer.innerText = greeting;
 // }
```

Comment:
If this method is going to be replaced.
You can add // TODO here.
Ex: // TODO: Replace it with XXX approach.